### PR TITLE
fixes the cybersun captain and the ammo types

### PR DESCRIPTION
**Pull request by ErwinSnowden**
## About The Pull Request
Fixes the captain to actually be playable without wasting a majority of their ammo to break open lockers. Re-adds their jumpsuit, might be the wrong jumpsuit. Fixes ammo types so they can actually fight.

## Why It's Good For The Game
I like playing as these roles and this change was not documented and I can't bother trying to track it down and revert it.

## Changelog
:cl:
fix: The Cybersun Captain now spawns with a jumpsuit again, not sure how that was removed.
fix: The Cybersun ship now contains usable ammo for the crew of the ship.
/:cl:

### CODE REVIEW

**Comment by EgorDinamit** [comment]
Okay, but as far as I can tell, captain's outfit copies stuff from the crew, so there is no need in adding jumpsuit here (Might be wrong, check current code).

File: code/game/objects/structures/ghost_role_spawners.dm [hunk 1]
```diff
@@ -697,6 +697,7 @@
 
 /datum/outfit/syndicatespace/syndicaptain
```
Comment (EgorDinamit) [inline code/game/objects/structures/ghost_role_spawners.dm:698]:
```suggestion
/datum/outfit/syndicatespace/syndicrew/syndicaptain
```

**Review by EgorDinamit — COMMENTED** [review]
Change outfit's name here and in the spawner's datum.
It will copy part of basic crew outfit.

File: code/game/objects/structures/ghost_role_spawners.dm (hunk 1 above)
Comment (EgorDinamit) [reply code/game/objects/structures/ghost_role_spawners.dm:698]: And replace outfit var with it in "/obj/effect/mob_spawn/human/syndicatespace/captain" part, at line 673.

**Comment by ErwinSnowden** [comment]
This would result in the captain getting extra gear, to note, night vision and an extra pistol. Not something I'm keen on doing because this is a fix, not a re-balance.

**Comment by EgorDinamit** [comment]
He is supposed to get everything basic crew gets, with an exception for pistol.
Just place Stechkin APS from the backpack into the left pocket, and it will be fine.

**Review by EgorDinamit — CHANGES_REQUESTED** [review]
Make captain's outfit copy crew outfit and place Stechkin APS in l_pocket, instead of the backpack.

**Comment by EgorDinamit** [comment]
I'll fix it in and add some other administrative features to Cybersun in #356

**Pull request closed without merging**